.github/workflows/trigger-nightly.yml

```diff
@@ -19,12 +19,14 @@ jobs:
           token: ${{ secrets.NIGHTLY_BUILD_GH_TOKEN }}
       -
         name: 'Push new tag'
-        # TODO determine nightly version based on latest minor version
         run: |
           git config user.name "${GITHUB_ACTOR}"
           git config user.email "${GITHUB_ACTOR}@users.noreply.github.com"
 
-          TAG=v0.2.0-nightly.$(date +'%Y%m%d')
+          DESCRIBE=`git describe --match "v[0-9].[0-9].[0-9]" --abbrev=0`
+          MINOR_VERSION=`echo $DESCRIBE | awk '{split($0,a,"."); print a[2]}'`
+          MINOR_VERSION="$((${MINOR_VERSION} + 1))"
+          TAG="v0.${MINOR_VERSION}.0-nightly.$(date +'%Y%m%d')"
```

Comment:
Since we don't know what will be last minor version in v0, we can't automate the switch to v1. However, we made our lives a bit easier by not having to worry about the minor versions pre-v1.

Reply:
I already made it dynamic.. but still trying to figure out the problem with `git describe`. I'll create a new PR for the changes when done.

Reply:
Don't worry about the major version, @hariso made a good point that we don't know when we're going to start targeting the release of `v1` so at that point we'll already have to change the script to start producing tags like `v1.0.0-nightly...`.